index.bs

```diff
@@ -1338,9 +1292,16 @@ The value of the \[[Prototype]] internal slot of the Module constructor is the i
 
 Besides the internal slots and the length property (whose value is 0), the Loader constructor has the following properties:
 
-<h4 id="Module.evaluate">Module.evaluate(m)</h4>
+<h4 id="Module.evaluate">Module.evaluate(ns)</h4>
 
-<b>TODO:</b> way to force evaluation of a module namespace exotic object (<code>Reflect.Module.evaluate(m)</code>? <code>m[Reflect.Module.evaluate]()</code>?)
+<emu-alg>
+1. If _ns_ is not a module namespace exotic object, throw a *TypeError* exception.
+1. Let _module_ be _ns_.[[Module]].
+1. Let _entry_ be _module_.[[Entry]].
+1. Let _status_ be EnsureEvaluated(_entry_).
+1. RejectIfAbrupt(_status_);
+1. Return a promise resolved with *true*.
```

Comment:
* probably should resolve to `undefined` because they already have access to the `ns`.
* returning a promise because in the future we might want to hold this until completion of execution (assuming top level away lands).